### DISCUSSION TITLE
Hmm.. messed up 'Originals'

### DISCUSSION THREAD

**Discussion by andrewlow** [photoprism/photoprism#989] [Q&A] [answered]
I've adopted the  approach of importing all my photos into Photoprism - vs. having it Index an existing tree.

Then I messed up and dragged some files into the Originals directory 

`./originals/2020/DSC_1098.JPG`

Of course, PhotoPrism gracefully dealt with this and has started to manage it.

But.. I want to fix this, and actually import the file.

No problem, right? I moved the file `./originals/2020/DSC_1098.JPG` into the `./import` directory and kicked off an import.

Sigh - PhotoPrism happily just 'moved' the file right back to the same place..

I also tried moving it out of the originals directory - kicking a re-index, then re-importing. Same result.

What am I doing wrong? Do I need to go do some database editing to really 'forget' the photo?

**Comment by lastzero** [comment, marked as answer]
The file is skipped while importing as it already exists. We currently don't move or rename existing originals. That may be added in a later release and it requires a lot of work and testing - easy to mess up a complete photo library with this.

**Reply by andrewlow** [reply]
Hmm. I was hopeful that by 'removing' the file I could add it again as if it were 'new' then get it imported correctly. 

Oh well, I can live with this until such time that you allow me to really 'delete' things from my library (and thus originals) and then I'll be able to re-import.

So -- following https://docs.photoprism.org/user-guide/organize/delete/ to 'archive' then remove the actual file from originals..

Will not result in it being really forgotten? (ie: if I re-import the deleted photo.. it will still be known and land in the same location)

**Reply by lastzero** [reply]
You may archive and then delete it completely - this will import the "new" file next time.

**Reply by lastzero** [reply]
![Screenshot 2021-02-01 at 22 05 09](https://user-images.githubusercontent.com/301686/106518051-9840da80-64d9-11eb-994c-fb7f42b85304.png)

**Reply by andrewlow** [reply]
This is probably a good reason to become a sponsor!
